Update npm to latest version in docker image

jorrit opened the pull request:
For me, the docker method of creating the function zip hangs during the `npm install --production` phase. I solved it by updating the npm tool to the latest version in the docker image.

jorrit committed:
Update npm to latest version in docker image

jpignata commented on Dockerfile:
```diff
@@ -9,7 +9,8 @@ RUN yum -y groupinstall "Development Tools" && \
     rpm --install --force ns.rpm && \
-    rm --force ns.rpm
+    rm --force ns.rpm && \
+    npm install -g npm@latest
```
Any chance you can reorder these to leave cleanup steps at the end?

jpignata commented:
Thanks! I can't reproduce this - but I don't see any harm in updating npm.

jorrit commented:
I thought it was weird too, I don't understand why this happens. A docker image should be the best way of ensuring consistency, but for me the npm install command never finishes, or I don't wait long enough. I have to kill the container with `docker kill`.

Anyway, I updated the commit and added `npm cache clean` after the install to make the image even cleaner.

jpignata commented:
Thanks for your contribution!
